### PR TITLE
v8 - uduf- Fixes 4742 - no assembly scanning for dashboards

### LINKED ISSUE #4742: v8 dashboards should not be found by assembly scanning
There's a couple of reasons for this:

* Consistency - currently sections built in c# need to be manually registered in the container so to be consistent, so should dashboards
* Performance - currently dashboards built in c# are assembly scanned which has an overhead on startup performance even though dashboards are not critical to the website operation

Both sections and dashboards can easily be registered in the package.manifest, this is only to do with dashboards registered via c# and if they are created by c# they should be manually added to the container in c# just like sections.

### PR DESCRIPTION
**Pull request by Shazwazza**
Fixes #4742

### CODE REVIEW

File: src/Umbraco.Web/Runtime/WebRuntimeComposer.cs [hunk 1]
```diff
@@ -238,7 +239,17 @@ public override void Compose(Composition composition)
...
+                .Add<ExamineDashboard>()
+                .Add<FormsDashboard>()
+                .Add<HealthCheckDashboard>()
+                .Add<ManifestDashboard>()
+                .Add<MediaDashboard>()
+                .Add<MembersDashboard>()
+                .Add<PublishedStatusDashboard>()
+                .Add<RedirectUrlDashboard>()
+                .Add<RedirectUrlDashboard>()
+                .Add<SettingsDashboard>();
```
Comment (bjarnef) [inline src/Umbraco.Web/Runtime/WebRuntimeComposer.cs]: @Shazwazza I guess it only need to add `RedirectUrlDashboard` once ;)

Comment (Shazwazza) [reply src/Umbraco.Web/Runtime/WebRuntimeComposer.cs]: Thanks :)

**Comment by Shazwazza** [comment]
@bergmania can we please make sure this is merged for netcore? I think this will no longer be accepted for v8 but it should be the case for netcore for sure

**Comment by bergmania** [comment]
@Shazwazza, Done.. 
I had to add `ProfilerDashboard` to the list

**Comment by Shazwazza** [comment]
@bergmania legend! ... i guess i can just close this then?

**Comment by bergmania** [comment]
> @bergmania legend! ... i guess i can just close this then?

I guess so.. This is too breaking for v8, IMO :)

**Pull request closed without merging**